cms/jinja2/templates/components/accredited/accredited-logo.html

```diff
@@ -1,3 +1,3 @@
 <a href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/" class="common-header__accreditation-link">
-    <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Statistics Authority Kitemark" class="{{ accredited_logo_classes }}" width="90">
+    <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Official Statistics Badge - Accredited" class="{{ accredited_logo_classes }}" width="90">
```

Comment:
The live site has some inconsistencies. What we have matches the live site's release calendar (but fails accessibility), but their article pages use the following alt text, which I wonder if we should use `These are accredited official statistics. Click for information about types of official statistics.`

On a related note, the accredited logos in Heros are links in the live site, but they are not when using the DS header as we haven't configured `officialStatisticsBadgeUrl` in the Hero component. Do you mind sorting that as part of these rounds of fixes (separate PR)?. Thanks